avm/res/storage/storage-account/local-user/main.bicep

```diff
@@ -57,6 +57,7 @@ output resourceId string = localUsers.id
 //   Definitions   //
 // =============== //
 
+@export()
 @secure()
 type sshAuthorizedKeysType = {
```

Comment:
I guess we should fix this type eventually - removing the nested `secureList`, use `@secure()` for the `key` property instead, make this type singular und adding the `[]` to the parameter instead. 

However, we do not have to do it now. Wanted to hear your thoughts though and if we don't do it now, create an issue so that we remember 💪

Reply:
Raised a ne issue #3640